.travis.yml

```diff
@@ -3,7 +3,11 @@ dist: trusty
 
 language: go
 go:
+    - 1.7.1
```

Comment:
Do we need these? Is it common to find problems with one minor version but not another?